webui/react/package.json

```diff
@@ -33,9 +33,9 @@
     "portable-fetch": "^3.0.0",
     "pretty-bytes": "^5.4.1",
     "query-string": "^6.13.5",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-helmet": "^6.1.0",
+    "react": "^17.0.2",
```

Comment:
There looks to be something off in the [LogViewer storybook](http://localhost:9009/?path=/story/logviewer--default) which is broken in this branch, but not on master?

Reply:
great find!

Turned out `LogViewer` is using `Page` which requires the new `HelmetProvider` as a decorator. Newest commit should fix this issue.